RevisionNotes/MIT_lec7.md

```diff
@@ -0,0 +1,59 @@
+## Computing Ax=0, Pivot variables, special solutions.
+
+* `pivot` = first non zero element in every row after elimination
+* `pivot columns` = columns containing pivot 
+* `pivot variables` = variables corresponding to pivot columns
+* `free columns` = columns without pivot
+* `free variables` = variables corresponding to free columns
+* `rank` = number of pivot variables
+
+## Computing Ax = 0 using echelon form.
+
+Consider the Martix A in the below diagram, using elementary row transformation converting the matrix in echelon form.
+
+![Echelon form](./Images/echelon_form_lect7.png)
+
+Now finding the solution x by putting free variables as random constants
+
+![Echelon solution](./Images/echelon_sol_lect7.png)
+
+## Reduced Row Echelon Form
+
+If the leading coefficient in each row is the only non-zero number in that column, the matrix is said to be in reduced row echelon form(rref). Consider the below example.
```

Comment:
Please mention the fact that leading non-zero coefficient must always be unity in rref form.